engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/ExecStrictCompilerTest.java

```diff
@@ -28,6 +30,8 @@ public class ExecStrictCompilerTest {
           .withModifiedContext(ctxBldr -> ctxBldr.option(RuntimeOptions.STRICT_ERRORS, "true"))
           .build();
 
+  @Rule public ReportLogsOnFailureRule appenderRule = new ReportLogsOnFailureRule();
```

Comment:
Any test that already uses `@Rule ContextUtils` could implement the ReportLogsOnFailure logic by just setting a default LogHandler. Right? Why do we need yet another rule for such tests?

Reply:
It could also be problematic that in many cases, we use `ContextUtils` as a `@ClassRule`, but `ReportLogsOnFailureRule` is always used as just `@Rule`. OK, let's keep them separated.